chain/cosmos/chain_node.go

```diff
@@ -703,7 +702,7 @@ func (tn *ChainNode) StartContainer(ctx context.Context) error {
 	tn.Container = c
 
 	port := dockerutil.GetHostPort(c, rpcPort)
-	tn.logger().WithField("container", tn.Name()).Infof("RPC => %s", port)
+	tn.logger().With("container", tn.Name()).Infof("RPC => %s", port)
```

Comment:
Using formatted messages misses part of the point of structured logging. This ought to provide a structured `port` field and log a fixed message like "RPC listening".

Reply:
Good point. This was the previous pattern.